api/src/pcapi/static/backofficev3/js/addons/pc-field-list.js

```diff
@@ -130,11 +131,40 @@ class PcFieldList extends PcAddOn {
     const $li = document.createElement('li')
     const $newEmptyForm = $emptyForm.cloneNode(true)
     $li.append($newEmptyForm, $emptyForm.cloneNode(true), $removeButton) // we keep a clone within the newly added and edited $newEmptyForm form
-    $li.querySelector(PcFieldList.FIELD_ELEMENT_BEARING_VALUE_SELECTOR).classList.add(formFieldClass)
+    // tomSelect code
+    const emptyTomSelectClass = this.app.addons[this.app.addons.PcTomSelectFieldId.constructor.ID].constructor.EMPTY_PC_TOM_SELECT_FIELD_CONTAINER_CLASS
+    const tomSelectClass = this.app.addons[this.app.addons.PcTomSelectFieldId.constructor.ID].constructor.PC_TOM_SELECT_FIELD_CLASS
+    $newEmptyForm.classList.remove(PcFieldList.PC_FORM_FIELD_EMPTY_CONTAINER_CLASS)
+    const $tomSelectEmptys =  $li.getElementsByClassName(emptyTomSelectClass)
+    for(let i = 0; i < $tomSelectEmptys.length; i++){
```

Comment:
```suggestion
    for(let i = 0; i < $tomSelectEmptys.length; i+=1){
```

Mais en ES7, c'est plutôt:

```suggestion
    tomSelectEmptys,forEach(($tomSelectEmptys) => {
       // ...
    })
```

Le 2nd paramètre de l'arrow function est également l'indice.

Reply:
Ça marche pas avec le `continue`

Reply:
True

Reply:
Sinon le continue fonctionne dans un `for (const ... of ...)` je crois